pay-api/migrations/versions/2024_02_13_b96e31f2f30e_add_refund_amount_payment_line_item.py

```diff
@@ -0,0 +1,24 @@
+"""empty message
+
+Revision ID: b96e31f2f30e
+Revises: 03884e4187de
+Create Date: 2024-02-13 16:27:37.024941
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'b96e31f2f30e'
+down_revision = '03884e4187de'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('payment_line_items', sa.Column('refund_amount', sa.Numeric(), nullable=True))
```

Comment:
This wont work because of the other columns:

```
pay-db=# \d payment_line_items
                                           Table "public.payment_line_items"
        Column         |          Type          | Collation | Nullable |                    Default
-----------------------+------------------------+-----------+----------+------------------------------------------------
 id                    | integer                |           | not null | nextval('payment_line_items_id_seq'::regclass)
 invoice_id            | integer                |           | not null |
 filing_fees           | numeric(19,2)          |           | not null |
 fee_schedule_id       | integer                |           | not null |
 quantity              | integer                |           |          |
 description           | character varying(200) |           |          |
 gst                   | numeric(19,2)          |           |          |
 pst                   | numeric(19,2)          |           |          |
 total                 | numeric(19,2)          |           | not null |
 line_item_status_code | character varying(20)  |           | not null |
 future_effective_fees | numeric(19,2)          |           |          |
 priority_fees         | numeric(19,2)          |           |          |
 waived_by             | character varying(50)  |           |          |
 waived_fees           | numeric(19,2)          |           |          |
 fee_distribution_id   | integer                |           |          |
 service_fees          | numeric(19,2)          |           |          |
```

Reply:
If we knew 100% the priority_fee, service_fee, future_effective_fee we're going to the same GL, maybe we could use another field eg. `refund_fee_amount`